mail_activity_board/models/mail_activity.py

```diff
@@ -1,31 +1,27 @@
+# Copyright 2016 David Juaneda - <djuaneda@sdi.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-# SDI
-# author: djuaneda@sdi.es
 from odoo import api, models, fields
-from odoo.tools.safe_eval import safe_eval
 
 
 class MailActivity(models.Model):
     _inherit = "mail.activity"
 
     res_model_id_name = fields.Char(
-        default="", related='res_model_id.name',
-        string="Origin", store=False, readonly=True)
+        related='res_model_id.name', string="Origin",
+        store=False, default="", readonly=True)
```

Comment:
`store=False` is the default, you can remove it.

Also, `default=""` can lead to confusing behavior, because in Odoo, the standard is that empty fields return `False` always. Having `""` instead of `False` is unconsistent, so please remove that too.

Reply:
Deleted store=False.
Deleted default="".